node-wit/node-wit-tests.ts

```diff
@@ -0,0 +1,14 @@
+import { Wit, log } from "node-wit";
+
+var wit = new Wit({accessToken: "", logger: new log.Logger(log.DEBUG)});
+wit.message("Hello", {}).then((res) => {
+    console.log(res._text);
+
+    for (let entity of res.entities) {
+        
+        console.log(entity.value + ": " + entity.confidence * 100 + "%");
```

Comment:
Since `entity.confidence` is optional, this may be a compile error if we get around to fixing Microsoft/TypeScript#12795.